traffic_ops/v4-client/endpoints.go

```diff
@@ -22,6 +22,6 @@ package client
 // Versions are ordered latest-first.
 func apiVersions() []string {
 	return []string{
-		"4.0",
+		"4.1",
```

Comment:
This should list both minor versions, not just the latest.

Reply:
Thanks for the explanation on slack.